README.md

```diff
@@ -250,16 +250,24 @@ Now Prettier will be run every time the file is saved.
 
 ### VSCode
 
-VSCode is not familiar with the solidity language, so [`solidity`](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) support needs to be installed.
+VSCode is not familiar with the solidity language.
+There are 2 extensions that you can install to provide support for solidity:
+
+- [`solidity`](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity)
+- [`hardhat-solidity`](https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity)
 
 ```Bash
 code --install-extension JuanBlanco.solidity
+# or
+code --install-extension NomicFoundation.hardhat-solidity
 ```
 
-This extension provides basic integration with Prettier for most cases no further action is needed.
+> :warning: **These 2 extensions offer similar functionality and will clash with each other**: Please choose which one matches your projects better.
+
+These extensions provides basic integration with Prettier and for most cases no further action is needed.
```

Comment:
how about:

These extensions provide basic integration with Prettier; in most cases, no further action is needed.